.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
Why do we need a specific java version now?

Reply:
Because it doesn't work against the latest.  I'll raise a ticket for investigating that.